parsec/mca/device/device_gpu.h

```diff
@@ -231,6 +231,11 @@ struct parsec_device_gpu_module_s {
                                                    *   is increased every time a new data is made available, so
                                                    *   that we know which tasks can be evaluated for submission.
                                                    */
+
+    int                         memory_percentage; /**< What % of the memory available on the device we want to use*/
+    int                         number_blocks;     /**< In case memory_percentage is not set, how many blocks we want to allocate on the device */
+    size_t                      eltsize;           /**< And what size in byte are these blocks */ 
```

Comment:
Indentation seems off by 1

```suggestion
    int                        memory_percentage; /**< What % of the memory available on the device we want to use*/
    int                        number_blocks;     /**< In case memory_percentage is not set, how many blocks we want to allocate on the device */
    size_t                     eltsize;           /**< And what size in byte are these blocks */ 
```

Reply:
see #616